eZ/Publish/Core/settings/search_engines/legacy/field_value_mappers.yml

```diff
@@ -0,0 +1,3 @@
+parameters:
+    ezpublish.search.common.field_value_mapper.date.class: eZ\Publish\Core\Search\Legacy\Content\FieldValueMapper\DateMapper
+    ezpublish.search.common.field_value_mapper.float.class: eZ\Publish\Core\Search\Legacy\Content\FieldValueMapper\FloatMapper
```

Comment:
Well... It looks like we don't. I must have misunderstood the idea behind the field value mappers when we started this [long time ago](https://github.com/ezsystems/ezpublish-kernel/pull/1660#issuecomment-223565729).

Initially I assumed that my `FullTextMapper` would use this to get a proper searchable value. 
However, what I needed has already been available via `eZ\Publish\SPI\FieldType\Indexable::getIndexData` and [I used it in `FullTextMapper`](https://github.com/alongosz/ezpublish-kernel/blob/c5746c1d1939b088840e8edb25dbc500f3395704/eZ/Publish/Core/Search/Legacy/Content/Mapper/FullTextMapper.php#L114), forgetting about initial consideration of field value mappers.

Right now, after looking over once again how field value mappers are used in Solr and Elasticsearch, I think we even didn't need to touch that to solve this issue. 

@pspanja: so I probably should remove those lines as well:
 - [eZ/Bundle/EzPublishLegacySearchEngineBundle/EzPublishLegacySearchEngineBundle.php#L31](https://github.com/alongosz/ezpublish-kernel/blob/c5746c1d1939b088840e8edb25dbc500f3395704/eZ/Bundle/EzPublishLegacySearchEngineBundle/EzPublishLegacySearchEngineBundle.php#L31).

- [eZ/Publish/API/Repository/Tests/SetupFactory/Legacy.php#L381](https://github.com/alongosz/ezpublish-kernel/blob/c5746c1d1939b088840e8edb25dbc500f3395704/eZ/Publish/API/Repository/Tests/SetupFactory/Legacy.php#L381).

- [eZ/Publish/Core/settings/containerBuilder.php#L67](https://github.com/alongosz/ezpublish-kernel/blob/c5746c1d1939b088840e8edb25dbc500f3395704/eZ/Publish/Core/settings/containerBuilder.php#L67).